file-formats/ilmb/type/zif_aff_ilmb_v1.intf.abap

```diff
@@ -0,0 +1,83 @@
+INTERFACE zif_aff_ilmb_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Condition Field</p>
+    "! Condition field
+    BEGIN OF ty_cond_field,
+      "! <p class="shorttext">Name</p>
+      "! Condition field
+      name             TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Condition Field Description</p>
+      "! Field name description
+      condition_field_description TYPE zif_aff_types_v1=>ty_description_60,
+      "! <p class="shorttext">Data Element</p>
+      "! Data element
+      data_element                TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Source Table</p>
+      "! Source table
+      source_table                TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Source Field</p>
+      "! Source field
+      source_field                TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">No Intervals</p>
+      "! No intervals
+      no_intervals                TYPE abap_bool,
+    END OF ty_cond_field,
+    "! <p class="shorttext">Condition Fields Details</p>
```

Comment:
Here, we don't need the 'Datails', only for the underlying structure of the table
```suggestion
    "! <p class="shorttext">Condition Fields</p>
```

Reply:
You are right, @wurzka. I guess this was my fault/suggestion. Sorry, I mixed it up.

Reply:
Have set it to Condition Fields